blivet/devices/lvm.py

```diff
@@ -1471,6 +1475,13 @@ def vg_space_used(self):
             space += Size(blockdev.lvm.get_thpool_padding(space, self.vg.pe_size))
         return space
 
+    def _pre_create(self):
+        if self._from_lvs:
+            # make sure all the LVs this LV should be created from exist
+            for lv in self._from_lvs:
+                if not lv.exists:
+                    lv.create()
```

Comment:
I think we should consider the tests a form of documentation and therefore require the same procedure when testing as in production. This is handy, but I think it is better to be consistent.